client/pages/courses/index.vue

```diff
@@ -373,13 +376,19 @@ export default {
             return this.$store.getters['planner/isInPlan'](this.courseInfo)
         },
         categories_results() {
-            const [availableFilter, campusFilter, subjectFilter, courseFilter, dayFilter, profFilter, attrFilter] = this.selected,
+            //TODO actual filtering
```

Comment:
I haven't checked if the filtering works just yet but we can probably remove this TODO comment since it's done.